java/openjdk/Portfile

```diff
@@ -268,6 +321,8 @@ subport openjdk11-openj9-large-heap {
 
 subport openjdk11-sap {
     # https://github.com/SAP/SapMachine/releases/
+    supported_archs  x86_64
```

Comment:
Change `# https://github.com/SAP/SapMachine/releases/` to `https://sap.github.io/SapMachine/latest/11`

Reply:
Let's do that via a separate PR and keep this one about adding Corretto.

Reply:
Ok